src/ast/mod.rs

```diff
@@ -1247,6 +1247,14 @@ pub enum Statement {
         /// deleted along with the dropped table
         purge: bool,
     },
+    /// DROP Function
+    DropFunction {
+        if_exists: bool,
+        /// One or more function to drop
+        func_desc: Vec<DropFunctionDesc>,
+        /// `CASCADE` or `RESTRICT`
+        option: Option<DropFunctionOption>,
```

Comment:
I will change it later, thanks for your suggestion

Reply:
> It might be possible to use https://github.com/sqlparser-rs/sqlparser-rs/blob/d420001c37eeb38aac26a85dd5661755d6ef4228/src/ast/ddl.rs#LL602C16-L602C16

done